owenhallgren/stretch-project
# Feature/contact pop up

## What does this PR do?

This adds a popup to the accept review button, the user will be briefed with more information before they fully accept the review. They will also be given instructions on what to do if they accept. 
## How should it be tested in the UI?

Click the accept review button and make sure the popup looks good and preforms as expected.
